src/dotdirdeps.cpp

```diff
@@ -155,6 +155,7 @@ static void drawDirectory(TextStream &t, const DirDef *const directory, const Do
       "fillcolor=\"" << getDirectoryBackgroundColor(directory->level()-startLevel)   << "\", "
       "color=\""     << getDirectoryBorderColor(property)                            << "\", "
       "URL=\""       << addHtmlExtensionIfMissing(directory->getOutputFileBase())    << "\""
+      "tooltip=\""   << directory->briefDescriptionAsTooltip() << "\""
```

Comment:
I would expect a similar operation as is done by`escapeTooltip()` for `DotNode` is needed before writing the result to avoid quotes in the description to cause invalid output.

Reply:
Done